{"name": "test_run1", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 200\n +  where 403 = <Response [403]>.status_code", "trace": "def test_run1():\n        HEADERS = {'Authorization': user_data.TOKEN}\n        PARAM = {'email': 'test-datatest11111211@yandex.ru'}\n        LINK = \"https://stellarburgers.nomoreparties.site/api/auth/user\"\n        RESPONCE = requests.patch(LINK, headers=HEADERS, data=PARAM)\n>       assert (RESPONCE.status_code) == 200\nE       assert 403 == 200\nE        +  where 403 = <Response [403]>.status_code\n\ntest\\changing_user_data\\test_with_auth.py:16: AssertionError"}, "start": 1694413277757, "stop": 1694413278571, "uuid": "8074b7b5-3fbd-4b34-aa85-b7dd7dd9f30b", "historyId": "23d64fe7ee8361a5fe0ff981a8489353", "testCaseId": "23d64fe7ee8361a5fe0ff981a8489353", "fullName": "test.changing_user_data.test_with_auth#test_run1", "labels": [{"name": "parentSuite", "value": "test.changing_user_data"}, {"name": "suite", "value": "test_with_auth"}, {"name": "host", "value": "DESKTOP-1SIQC8C"}, {"name": "thread", "value": "6016-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.changing_user_data.test_with_auth"}]}